Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.6.2-alpine
+
+RUN apk add --update git && apk add --update make && rm -rf /var/cache/apk/*
+
+ADD . /go/src/github.com/ernestio/network-creator-aws-connector
```

Comment:
The correct approach should be using environment replacement, so we should look at something like
```
ADD . /go/src/github.com/${GITHUB_ORG:-ernestio}/network-creator-aws-connector
```
So in case we run docker on an environment with `GITHUB_ORG` defined the resulting container will get the source from that organisation instead of the default "ernestio". Even this is still a limitation for the repo name, I think it covers our needs atm.

Ref: https://docs.docker.com/engine/reference/builder/#/environment-replacement